app/main/actions/open.js

```diff
@@ -0,0 +1,9 @@
+const electron = require('electron');
+
+const { shell } = electron;
```

Comment:
I think this is superfluous to be honest.
you might as well in L7: `electron.shell.openItem(arg)`

At least the context of the call is clear.

---

sidenote: yes I'm back from the dead